docs/how-to/deploy/deploy-to-production.mdx

```diff
@@ -132,3 +132,47 @@ To achieve high throughput and low latency:
 - **Disable Decision Debug Info**: In production, you can turn off decision debug information to enhance performance. Disable debug logging by adding `--env PDP_DEBUG=False` to the PDP command line.
 - **Optimize Base Policy**: Tailor the base policy to your specific use case. For example, disabling ReBAC checks (if not used) can speed up policy execution (this requires our tech support).
   With these optimizations, you can expect several thousand RPS with less than 10ms latency per request.
+
+
+## PDP Offline Mode
+
+The PDP relies on Permit's cloud for fetching configuration, loading policies, and keeping those up to date.
+Despite the high availability of Permit's cloud, there still might be cases when the PDP can't reach it.
+For extra resilience, the PDP can be configured to work in offline mode, where it will use a local backup of the configuration and policies.
+
+### How it works
+When the PDP is started in offline mode, it will try to fetch the configuration from Permit's cloud as usual.
```

Comment:
> When the PDP is started in offline mode
You mention this but discuss how to do so only in the following section... maybe switch the order ?

Reply:
Alright - so maybe I'll first have the section oh how to run it simply - and then "How it works" - and then the advanced configuration.

Reply:
@orweis tell me what you think now